packages/components/src/components/notification/notification.stories.ts

```diff
@@ -3,21 +3,27 @@
 import '../../solid-components';
 import { html } from 'lit';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 
 const { argTypes, parameters } = storybookDefaults('sd-notification');
 const { generateTemplate } = storybookTemplate('sd-notification');
 const { overrideArgs } = storybookHelpers('sd-notification');
 
+/**
+ * Used to communicate important information or status to the user. Notifications can be displayed inline or as a toast / toast stack.
+ *
+ *  **Related templates**:
+ * - [Notifications](?path=/docs/templates-notifications--docs)
```

Comment:
In Figma we have "Notification" here "Notifications".
Please align with Design

Reply:
Updated on storybook